.github/workflows/check-links.yml

```
@@ -27,6 +27,8 @@ jobs:
           --exclude "^https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$"
           --exclude "^https://download1.rpmfusion.org/free/el/rpmfusion-free-release-$"
           --exclude "^http://maps.google.com/mapfiles/kml/$"
+          --exclude "^http://aa.usno.navy.mil/faq/docs/lawyers.php"
+          --exclude "^Uhttp://my.server.com/images"
```

Better to add the `$` sign to match the end of the string.
```suggestion
          --exclude "^http://aa.usno.navy.mil/faq/docs/lawyers.php$"
          --exclude "^Uhttp://my.server.com/images$"
```